.env-example

```diff
@@ -13,7 +13,10 @@ DJANGO_DEFAULT_THROTTLE_RATE_USER='120/minute'
 
 DJANGO_SECRET_KEY='secret key'
 
-DATABASE_URL='postgres:///test'
+DATABASE_NAME='drf'
```

Comment:
On .env-example, use descriptive values instead of the actual values for example: `DATABASE_NAME=your database name`

Reply:
You could also add comments to describe what the variable is going to be used for or how it will affect the application.